Tests/Marketplace/search_and_uninstall_pack.py

```diff
@@ -49,6 +49,57 @@ def get_all_installed_packs(client: demisto_client, unremovable_packs: list):
         return None
 
 
+def uninstall_all_packs_one_by_one(client: demisto_client, hostname, unremovable_packs: list):
+    """ Lists all installed packs and uninstalling them.
+    Args:
+        unremovable_packs: list of packs that can't be uninstalled.
+        client (demisto_client): The client to connect to.
+        hostname (str): cloud hostname
+
+    Returns (list, bool):
+        A flag that indicates if the operation succeeded or not.
+    """
+    packs_to_uninstall = get_all_installed_packs(client, unremovable_packs)
+    logging.info(f'Starting to search and uninstall packs in server: {hostname}, packs count to '
+                 f'uninstall:{len(packs_to_uninstall)}')
+    uninstalled_count = 0
+    failed_to_uninstall = []
+    if packs_to_uninstall:
+        for pack_to_uninstall in packs_to_uninstall:
+            logging.info(f"Attempting to uninstall a pack: {pack_to_uninstall}")
+            if uninstall_pack(client, pack_to_uninstall):
+                uninstalled_count += 1
+            else:
+                failed_to_uninstall.append(pack_to_uninstall)
+    logging.info(f"Finished uninstalling - Succeeded:{uninstalled_count} out of {len(packs_to_uninstall)}")
+    if failed_to_uninstall:
+        logging.error(f"Failed to uninstall:{','.join(failed_to_uninstall)}")
+    return uninstalled_count == len(packs_to_uninstall)
```

Comment:
What if there are some `unremovable_packs` in the installed packs?
(It isn't seems to be used in `get_all_installed_packs`)

Reply:
@tkatzir it's being handled by `get_all_installed_packs` could you look again at it.

Reply:
Indeed. I must have missed it.